plugins/modules/ovirt_host_network.py

```diff
@@ -244,10 +244,11 @@
 def get_bond_options(mode, usr_opts):
     MIIMON_100 = dict(miimon='100')
     DEFAULT_MODE_OPTS = {
+        '0': MIIMON_100,
         '1': MIIMON_100,
         '2': MIIMON_100,
         '3': MIIMON_100,
-        '4': dict(xmit_hash_policy='2', **MIIMON_100)
+        '4': dict(xmit_hash_policy='2')
```

Comment:
Shouldn't we align with the real bond mode numbers?

https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/administration_guide/sect-network_bonding#Bonding_Modes

Is there any official doc around bond mode default options in oVirt/RHV?

Reply:
> > Shouldn't we align with the real bond mode numbers?
> 
> @mwperina , can you please explain?

Here are only 5 modes out of 7 existing, so shouldn't we have a full list here?

> 
> > Is there any official doc around bond mode default options in oVirt/RHV?
> 
> No, on REST-API there might be no magic done by RHV at all, and on the host the local defaults apply, but are not reported by RHV. On webadmin, the logic from
> https://github.com/oVirt/ovirt-engine/blob/50129b39cd9315488e3da37446aee7aa65c0d71e/backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/network/BondMode.java#L45
> 
> is applied.

OK, so does that mean that we don't need to append `miimon='100'` inside ansible module, because engine performs that automatically? I think all defaults should be handled on engine side and clients likes SDKs and Ansible modules shouldn't have anything copied inside (like here)

Reply:
I did not need to add the other modes because they don't have attached `miimon='100'`.

Reply:
The MIIMON_100 is added by Engine UI, so either the ansible module could add it, too to be consistent with the Engine UI, or not to add it, to be consistent with the REST API.